### PR TITLE
www-servers/uwsgi: fix python 3.10

### PR DESCRIPTION
**Pull request by t0b3**
Closes: https://bugs.gentoo.org/829204

### CODE REVIEW

**Comment by t0b3** [comment]
https://github.com/gentoo/gentoo/commit/ef2531d06ec0220933c13f4ddc7491de4845dc16

**Comment by thesamesam** [comment]
The patches weren't applied, it seems, so should we do that?

**Comment by t0b3** [comment]
> The patches weren't applied, it seems, so should we do that?

True, I think this would help, yes...

File: www-servers/uwsgi/files/uwsgi-2.0.19.1-py310-fix.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+From c8c4bd1b5439217f2cb2f146caf162de69638bc1 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline www-servers/uwsgi/files/uwsgi-2.0.19.1-py310-fix.patch:1]: Remember to include links to where patches come from at the top.

**Pull request closed without merging**